ui/packages/consul-ui/mock-api/v1/internal/ui/service-topology/_

```diff
@@ -104,7 +104,7 @@ ${
       "ChecksPassing":${fake.random.number({min: 1, max: env('CONSUL_CHECK_COUNT', fake.random.number(10))})},
       "ChecksWarning":${fake.random.number({min: 0, max: env('CONSUL_CHECK_COUNT', fake.random.number(10))})},
       "ChecksCritical":${fake.random.number({min: 0, max: env('CONSUL_CHECK_COUNT', fake.random.number(10))})},
-      "Source": "${fake.helpers.randomize(['proxy-registration', 'specific-intention', 'default-allow', 'wildcard-intention'])}",
+      "Source": "${fake.helpers.randomize(['routing-config', 'proxy-registration', 'specific-intention'])}",
```

Comment:
Do we not receive a `wildcard-intention` value from Consul anymore?

Reply:
oh, no. I was playing with the mock data to get more `routing-config` cards. Fixing this.